src/Mod/CAM/PathSimulator/AppGL/GuiDisplay.cpp

```diff
@@ -31,26 +31,28 @@
 
 using namespace MillSim;
 
+// clang-format off
 GuiItem guiItems[] = {
```

Comment:
```suggestion
// clang-format off
// NOLINTBEGIN(*-magic-numbers)
GuiItem guiItems[] = {
```

Here the magic numbers are magic, and we cannot do much about it I think until proper layout is implemented we should simply disable these warnings here.